apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.html

```diff
@@ -783,6 +792,7 @@ <h5 class="pd-b05 font-15">
                 width="100%"
                 height="100%"
                 [src]="sanitizedPdfFileUrl"
+                title="sanatized PDF"
```

Comment:
Title are used by screen reader so sanitized is not required

Reply:
done